test/test_jinja2.py

```diff
@@ -0,0 +1,9 @@
+def test_svg(selenium):
```

Comment:
```suggestion
def test_jinja2(selenium):
```

Reply:
oops :smile:

Reply:
Fixed that and also added a line to `CHANGELOG.md`.